workload/workloads/userProfile.go

```diff
@@ -97,6 +143,53 @@ func createQueryIndex(collection *gocb.Collection) error {
 	return nil
 }
 
+func CreateFtsIndexes(cluster *gocb.Cluster) error {
```

Comment:
I wonder if this should create and ensure the index is ready?  Basically, with a large set of users, we don't want the index creation to still be happening during the load.  Maybe this is something to consider later.

Reply:
Could consider calling it something like EnsureFtsIndexes to be clear it's doing more than create, even if we add the build/check later.